ui-tests/cypress/support/pageObjects/pages/designer/widgetconfigs/ContentWidgetConfigPage.js

```diff
@@ -27,50 +27,27 @@ export class ContentListSelectModal extends Dialog {
               .children('td').eq(0)
               .children('input[type=checkbox]');
   }
-
-  getChooseButton() {
-    return this.getFooterActionByLabel('Choose');
-  }
 }
 
 export default class ContentWidgetConfigPage extends WidgetConfigPage {
   
   getAddContentButton() {
-    return this.getMainContainer().contains('Add existing content');
-  }
-
-  getChangeContentButton() {
-    return this.getMainContainer().contains('Change content');
-  }
-
-  getContentListTable() {
-    return this.getBody()
-      .children('.Contents__body')
-      .children(htmlElements.div).eq(1)
-      .children(htmlElements.div)
-      .children('.Contents__table')
-      .children(htmlElements.table);
+    return this.getInnerPanel().find('button.btn.btn-primary')
+      .contains(/^Add existing content$/);
```

Comment:
Avoid using contains to describe page structure

Reply:
here is also being added specificity with `button.btn.btn-primary` to select the exact description of the button (plus the label being exactly "Add existing content" with regex) to return. this is because on how this widget config elements have been structured. there's too much similar elements into this element panel.

Reply:
This seems to be an issue with the _app-builder_ code, we should have more reliable identifier for testing.
Anyway, same as above: if you cannot find any other mean to identify the element leave it, we'll improve it later, but at least put a comment on it (TODO or FIXME) to remind us it's a weak spot.

Reply:
yup we should add test id attributes for the element. for now I'll add TODO comment